demos/portal-embed-demo/README.md

```diff
@@ -0,0 +1,148 @@
+
+### 4. Play around with it ;-)
+😎😎😎
+<br/>
+<br/>
+## Explainer
+This is to show how you can use Portals in cross origin situations. The demo runs a two local express servers for PORTALOG and TTT Archive to simulate cross origin use case. By default, PORTALOG runs in [http://localhost:3000/](http://localhost:3000/) and TTT Archive runs in [http://localhost:3001/](http://localhost:3001/).
+
+The basic structure of the demo is explained below.
+
+![explainer](https://cdn.glitch.com/98449704-33d8-49b2-88f2-aa6d2aeba5d3%2Fportal_explainer.png?1556377936083)
+
+Before getting into details... this could be the current experience w/o Portals
+![withoutportals](https://cdn.glitch.com/98449704-33d8-49b2-88f2-aa6d2aeba5d3%2Fw_o_portals_new.gif?1556394769252)
+- You can embed thrid party contents with **iframes**.
+- But if you want to visit to the content, a browser navigation starts and it needs to render all the content again which often leads to **slow page load experience**.
+- Also in this case even if you are playing the audio, **it just stops** due to page navigation.
+<br/>
+
+**...But with having Portals** 🚪🏃💨
+![withportals](https://cdn.glitch.com/98449704-33d8-49b2-88f2-aa6d2aeba5d3%2Fw_portals.gif?1556394385809)
+
+You can embed third party content just like an `iframe`.
+```html
+<!-- You can use it like iframes -->
+<portal src='https://example.com'></portal>
+```
+OR
+```javascript
+// From JavaScript
+const portal = document.createElement('portal');
+portal.src = 'https://example.com';
+```
+> Demo code reference: [creating a Portal element](https://github.com/WICG/portals/blob/master/demos/portal-embed-demo/public/js/portalog/portals-controller.js#L37) 
+
+For the Portal content side, you can judge if the page is used inside a Portal and customize the UI.
+```javascript
+// Check if window.portalHost is available
+if(window.portalHost){
+   // Customize the UI when being
+   // embedded as portals
+}
+```
+> Demo code reference: [Check if window.portalHost is avaiable](https://github.com/WICG/portals/blob/master/demos/portal-embed-demo/public/js/ttt/portals-controller.js#L198) and [change the style](https://github.com/WICG/portals/blob/master/demos/portal-embed-demo/public/js/ttt/portals-controller.js#L200)
+
+For now, Portal will not process user input. If you want to interact with the Portal (like playing the audio in the demo), use `postMessage`/`onmessage`.
+```javascript
+// Send message from the portal element
+const portal = document.querySelector('portal');
+portal.postMessage({someKey: someValue}, ORIGIN);
+
+// Receive message via window.portalHost
+window.portalHost.addEventListener('message', evt => {
+   const data = evt.data;
+   // do the actual following
```

Comment:
This comment makes sense in the original context, but seems a little out of place here.

Reply:
Thank you so much! Yes, this is not good. Fixed it to "handle the event"
https://github.com/WICG/portals/pull/105/commits/192e74c821a6f2de6dfbfc61d591a3da2035d55c#diff-75118edbb0db728bc0648cb6e01c9c2bR91